docs/official.rst

```diff
@@ -44,50 +44,6 @@ Options
   * If none are inputed, the package ``qtX.Y.Z-essentials`` is downloaded (default Qt install, includes ``qtcreator``...)
   * If ``all`` is inputed, the package ``qtX.Y.Z-full`` is downloaded (includes everything)
 
-.. dropdown:: Click to see all the special packages (for Linux Qt 6.8.1)
-
-   .. code-block:: text
-
-      Name: qt6.8.1-essentials
-      Display name: Qt 6.8.1 Linux x86_64 Essential Components
-      Description: Qt 6.8.1 Linux x86_64 Essential Libraries, Headers, and Tools
-      Version: 6.8.1
-      Components: qt.qt6.681.linux_gcc_64
-       
-      Name: qt6.8.1-essentials-dev
-      Display name: Qt 6.8.1 Linux x86_64 Essential Components (dev)
-      Description: Qt 6.8.1 Linux x86_64 Essential Libraries, Headers, and Tools (dev)
-      Version: 6.8.1
-      Required aliases: qt6.8.1-essentials
-       
-      Name: qt6.8.1-full
-      Display name: Qt 6.8.1 Linux x86_64 All Components with Sources
-      Description: Qt 6.8.1 Linux x86_64 All Libraries, Headers, Tools, and Sources
-      Version: 6.8.1
-      Components: qt.qt6.681.src,extensions.qtwebengine.681.src
-      Required aliases: qt6.8.1-essentials,qt6.8.1-addons,qt6.8.1-extensions
-      Optional components: extensions.qtinsighttracker.681.src
-       
-      Name: qt6.8.1-full-dbg
-      Display name: Qt 6.8.1 Linux x86_64 All Components with Sources and Debug Information Files
-      Description: Qt 6.8.1 Linux x86_64 All Libraries, Headers, Tools, Sources, and Debug Information Files (dev)
-      Version: 6.8.1
-      Components: qt.qt6.681.debug_info.linux_gcc_64,qt.qt6.681.debug_info,extensions.qtwebengine.681.debug_information
-      Required aliases: qt6.8.1-full-dev
-       
-      Name: qt6.8.1-full-dev
-      Display name: Qt 6.8.1 Linux x86_64 All Components with Sources (dev)
-      Description: Qt 6.8.1 Linux x86_64 All Libraries, Headers, Tools, and Sources (dev)
-      Version: 6.8.1
-      Required aliases: qt6.8.1-full
- 
-      Name: qt6.8.1-sdk
-      Display name: Qt 6.8.1 Linux x86_64 SDK
-      Description: Qt 6.8.1 Linux x86_64 SDK Tools (Qt Creator, Ninja, and CMake)
-      Version: 6.8.1
-      Components: qt.tools.qtcreator_gui,qt.tools.cmake,qt.tools.ninja
-      Required aliases: qt6.8.1-full-dev,qt6.8.1-full-dbg
-
```

Comment:
Was not displayed in the docs, as `dropdown` is not a thing in `.rst`. Moreover there is a link to the Qt website with this data somewhere else in the page